modules/swagger-codegen/src/main/resources/rust/model.mustache

```diff
@@ -0,0 +1,57 @@
+{{>partial_header}}
+{{#models}}
+{{#model}}
+{{#description}}
+// {{{classname}}} : {{{description}}}
+{{/description}}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct {{classname}} {
+{{#vars}}
+  {{#description}}
+  // {{{description}}}
+  {{/description}}
+  #[serde(rename = "{{baseName}}")] {{name}}: {{^required}}Option<{{/required}}{{{datatype}}}{{^required}}>{{/required}}{{#hasMore}},{{/hasMore}}
```

Comment:
I assume there's no sense to have an optional array as it's same as empty array in swagger semantics? Then arrays shouldn't be wrapped in `Option<>`.

Also: see next one